apps/transport/lib/transport_web/router.ex

```diff
@@ -55,6 +55,13 @@ defmodule TransportWeb.Router do
     )
   end
 
+  if Mix.env() == :dev do
```

Comment:
Note: l'usage de `Mix.env()` peut poser des soucis (https://stackoverflow.com/questions/51870979/why-does-mix-env-become-nil-on-production-when-being-called-in-a-layout) mais là l'évaluation se fait à compile-time donc je pense qu'on est bons.